waspc/data/Generator/templates/sdk/wasp/server/crud/_operationTypes.ts

```diff
@@ -91,7 +91,10 @@ export type GetQueryResolved = typeof _waspGetQuery
 
 {=# crud.operations.Create =}
 {=^ overrides.Create.isDefined =}
-type CreateInput = Prisma.{= crud.entityUpper =}CreateInput
+type CreateInput = Prisma.XOR<
```

Comment:
Fixes `CreateInput` to match what Prisma Client expects.

Reply:
I copied the type the `create` method has defined for its `data` input. Before I only use the simpler type because I thought it was enough, but now I realise it's better to follow 1:1 what Prisma uses internally - users will get the same type as if they were using `create` directly themselves.